modules/ROOT/pages/cloudhub/cloudhub-connector-ref.adoc

```diff
@@ -7,7 +7,7 @@ Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#
 
 CloudHub Connector v1.0
 
-Anypoint Connector for CloudHub provides the capability of Create and List Notifications, and to retrieve the status of created applications on CloudHub.
+Anypoint Connector for CloudHub enables you to create and list notifications, and to retrieve the status of created applications that are deployed to CloudHub.
```

Comment:
```suggestion
Anypoint Connector for CloudHub enables you to create and list notifications and to retrieve the status of created applications that are deployed to CloudHub.
```

Reply:
Following edits for parallelism:
<blah> | <verb phrase (configures, specifies, enables, rejects....